IntegrationWithPharmacies/UrgentMedicineOrderApi/Model/Entity.cs

```diff
@@ -0,0 +1,13 @@
+﻿
+namespace UrgentMedicineOrderApi.Model
+{
+    public class Entity
+    {
+        public Entity() { }
+        public Entity(int id)
+        {
+            this.id = id;
+        }
+        public int id { get; set; }
```

Comment:
Change name of property "id" into "Id".
Remove "this" from constructor and change it into "Id = id".